File: src/main/java/app/askresume/api/resume/mapper/CareerYearMapper.java
package app.askresume.api.resume.mapper;

import org.springframework.stereotype.Component;

@Component
public class CareerYearMapper {

    public String toCareer(int careerYear) {
        if (careerYear == 0) return "newcomer";
        else if (careerYear == 10) return "more than 10 years";
        else return careerYear + "year";
    }
}

File: src/main/kotlin/app/askresume/api/resume/mapper/CareerYearMapper.kt
package app.askresume.api.resume.mapper

import org.springframework.stereotype.Component

@Component
class CareerYearMapper {

    fun toCareer(careerYear: Int): String {
        return when (careerYear) {
            0 -> "newcomer"
            10 -> "more than 10 years"
            else -> "${careerYear}year"
        }
    }

}

